content/blog/posts/o3de-ux-past-present-future.md

```diff
@@ -0,0 +1,95 @@
+
+In contrast to the User Journey Map above, the phases below note how a user works on the Game Design workflow in Week 1 as well as the Team Setup in Month 1. According to [Rev the Engine](https://youtu.be/oFeqw-0tsE4) user research, users typically begin their gameplay, actor, work, and look development before collaborating with their team.
+
+![O3DE FTUE journey duration image](/images/blog/bjds-posts/03-ftue-journey-duration.jpg)
+
+### It's always Day 1
+
+Throughout the past year, I have observed a "Day 1" mentality here at Amazon. With this perspective, we're able to make high quality, high velocity decisions while improving through constant experimentation. In a similar way, a user progressing through O3DE's Day 1 workflow is concentrated on making smart (and fast) considerations while experimenting with the engine.
+
+>"_It means being brave enough to fail if it means that by applying lessons learnt, we can better surprise and delight customers in the future._"&mdash;[**Daniel Slater**](https://aws.amazon.com/executive-insights/content/how-amazon-defines-and-operationalizes-a-day-1-culture/)
+
+A user's first day typically consists of the Installation and Project Setup, or the user onboarding experience. User onboarding primarily involves users manoeuvring through a product so they can achieve results faster while trying to understand the product's value. In this case, the game engine.
+
+To learn more about our users' understanding of their Day 1 experience, we carried out numerous studies by interviewing, surveying, and testing over 40 users with varied backgrounds. As a result of our research, we were able to identify the typical behavioural patterns of our users. This allowed us to propose areas of improvement&mdash;the Project Manager and Gem Catalog.
+
+At this point, I worked with the UX team to redesign the Project Manager and Gem Catalog. For those who are new to this, Gems are packages of code or assets that augment game or simulation projects, and the Gem Catalog is a library of Gems that are native to O3DE. We conducted user interviews to understand the overall experience of the Gem system, any suggestions and issues, and the frequency with which users configure their project with Gems.
+
+>"_One of our biggest problems is not knowing what Gems are given to us, when installing or upgrading the application._"&mdash;**O3DE Research Participant**
+
+We found that it was difficult for new users to wrap their mind around the concept of Gems and their use within projects. With 80 Gems native to O3DE, the typical usage varies over two patterns—users either retain the default configuration throughout their development process or they enable all Gems to shorten build and compilation time. It was important to establish an information hierarchy, because of the Gems' low ==findability== and ==discoverability==.
```

Comment:
It was important to establish an information hierarchy, because of the Gems' low findability and discoverability.

(remove the == signs from the text)